KS.Fiks.IO.Arkiv.Client/Schema/arkivmeldingOppdatering.xsd

```diff
@@ -0,0 +1,263 @@
+            <xs:element name="merknadRegistrertAv" type="n5mdk:merknadRegistrertAv" minOccurs="0"/>
+            <xs:element name="skjermetObjekt" type="n5mdk:skjermetObjekt" minOccurs="0"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="merknadSlett">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="klassifikasjonOppdateringer">
+        <xs:sequence>
+            <xs:element name="oppdatering" type="klassifikasjonOppdatering" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="slett" type="klassifikasjonSlett" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="ny" type="arkivmelding:klassifikasjon" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+    
+    <xs:complexType name="klassifikasjonOppdatering">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+            <xs:element name="klasseID" type="n5mdk:klasseID" minOccurs="0"/>
+            <xs:element name="tittel" type="n5mdk:tittel" minOccurs="0"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="klassifikasjonSlett">
+        <xs:sequence>
+            <xs:element name="systemID" type="n5mdk:systemID"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="matrikkelnummerOppdateringer">
+        <xs:sequence>
+            <xs:element name="ny" type="arkivmelding:matrikkelnummer" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="slett" type="matrikkelnummerSlett" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="matrikkelnummerSlett"> <!-- her er vel alle til sammen nøkkelen for å slå opp. Eller burde det være mulig med en systemID her også? -->
+        <xs:sequence>
+            <xs:element name="kommunenummer" type="n5mdk:kommunenummer"/>
+            <xs:element name="gardsnummer" type="n5mdk:gardsnummer"/>
+            <xs:element name="bruksnummer" type="n5mdk:bruksnummer"/>
+            <xs:element name="festenummer" type="n5mdk:festenummer" minOccurs="0"/>
+            <xs:element name="seksjonsnummer" type="n5mdk:seksjonsnummer" minOccurs="0"/>
+        </xs:sequence>
+    </xs:complexType>
+    
+    <xs:complexType name="byggidentOppdateringer">
+        <xs:sequence>
+            <xs:element name="Oppdatering" type="byggidentOppdatering" minOccurs="0" maxOccurs="unbounded"/> <!-- er det naturlig å endre dette? -->
+            <xs:element name="Slett" type="byggidentSlett" minOccurs="0" maxOccurs="unbounded"/> <!-- skal man kunne slette? -->
+            <xs:element name="Ny" type="arkivmelding:byggident" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <!-- Trengs denne? Eller skal man bare kunne legge til ny og slette? -->
```

Comment:
Trengs denne?

Reply:
Samme for denne som for punkt og matrikkelnummer. Kanskje alle kun skal ha slett og legg til. Så lenge det ikke er noen systemID på denne må man uansett angitt alt gammelt. Men det kan hende at alle arkivleverandører har en systemID og at det derfor er greit å legge på denne også på disse objektene.

Reply:
Jeg la på en systemId